controllers/renovate_util.go

```diff
@@ -29,33 +30,54 @@ import (
 )
 
 const (
-	RenovateImageEnvName    = "RENOVATE_IMAGE"
-	DefaultRenovateImageUrl = "quay.io/konflux-ci/mintmaker-renovate-image:cdbc220"
-	DefaultRenovateUser     = "red-hat-konflux"
-	CaConfigMapLabel        = "config.openshift.io/inject-trusted-cabundle"
-	CaConfigMapKey          = "ca-bundle.crt"
-	CaFilePath              = "tls-ca-bundle.pem"
-	CaMountPath             = "/etc/pki/ca-trust/extracted/pem"
-	CaVolumeMountName       = "trusted-ca"
+	RenovateImageEnvName                    = "RENOVATE_IMAGE"
+	DefaultRenovateImageUrl                 = "quay.io/konflux-ci/mintmaker-renovate-image:083371d"
+	DefaultRenovateUser                     = "red-hat-konflux"
+	CaConfigMapLabel                        = "config.openshift.io/inject-trusted-cabundle"
+	CaConfigMapKey                          = "ca-bundle.crt"
+	CaFilePath                              = "tls-ca-bundle.pem"
+	CaMountPath                             = "/etc/pki/ca-trust/extracted/pem"
+	CaVolumeMountName                       = "trusted-ca"
+	BranchPrefix                            = "konflux/component-updates/"
```

Comment:
should we have word nudge instead of update here?

Reply:
it was like that before : 		
BranchName:         fmt.Sprintf("konflux/component-updates/%s", buildResult.Component.Name),

but I can change it if you want